extensions/amp-access/0.1/amp-access.js

```diff
@@ -145,30 +152,52 @@ export class AccessService {
   }
 
   /**
+   * @return {!Promise<string>}
+   * @private
+   */
+  getReaderId_() {
+    if (!this.readerIdPromise_) {
+      // TODO(dvoytenko): Confirm no consent is needed. The assumtpion is that
```

Comment:
Please remove the speculation from the TODO. Lets just file an issue to determine whether we need a way to pass in a consent promise. In any case it would be optional to do.